.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
```

Comment:
With a `.gitignore` file, you can avoid committing files you don't want saved in version control like this `.DS_Store` file and the generated directories below.